Makefile

```diff
@@ -412,10 +455,11 @@ clean:
 ##########
 
 define stop_noobaa
-	@echo "\033[1;34mStopping/removing test container\033[0m"
-	$(CONTAINER_ENGINE) network disconnect noobaa-net noobaa_$(GIT_COMMIT)_$(NAME_POSTFIX)
-	$(CONTAINER_ENGINE) stop noobaa_$(GIT_COMMIT)_$(NAME_POSTFIX)
-	$(CONTAINER_ENGINE) rm noobaa_$(GIT_COMMIT)_$(NAME_POSTFIX)
+	$(eval noobaa_container_name := $(if $(1),$(1),noobaa_$(GIT_COMMIT)_$(NAME_POSTFIX)))
```

Comment:
is it needed because in this case the image is named `noobaa-$(GIT_COMMIT)-$(NAME_POSTFIX)` and not `noobaa_$(GIT_COMMIT)_$(NAME_POSTFIX)`. why not just name it with `_`?